src/Nethermind/Nethermind.TxPool/ITxPoolConfig.cs

```diff
@@ -10,6 +10,9 @@ public interface ITxPoolConfig : IConfig
     [ConfigItem(DefaultValue = "5", Description = "The average percentage of transaction hashes from persistent broadcast sent to a peer together with hashes of the last added transactions.")]
     int PeerNotificationThreshold { get; set; }
 
+    [ConfigItem(DefaultValue = "70", Description = "Minimal percentage of the current base fee which must be surpassed by MaxFeePerGas for the transaction to be broadcasted.")]
```

Comment:
```suggestion
    [ConfigItem(DefaultValue = "70", Description = "The minimal percentage of the current base fee that must be surpassed by max fee (`max_fee_per_gas`) for the transaction to be broadcasted.")]
```
I don't think that our C# property name `MaxFeePerGas` is meaningful in the docs, so I changed it to the more common name of [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559#abstract). Please let me know if I'm wrong.